### ISSUE TITLE
MonetDB/e crashes on Windows with PreparedStatements and NULL values

### ISSUE THREAD

**Issue by rcblanke** [MonetDBSolutions/MonetDBe-Java#1]
MonetDB/e crashes on Windows with PreparedStatements and NULL values. 
When running the 
* ComplexTypes.java (https://github.com/MonetDBSolutions/MonetDBe-Java/blob/master/example/ComplexTypes.java), or
* PreparedQueries.java (https://github.com/MonetDBSolutions/MonetDBe-Java/blob/master/example/PreparedQueries.java)
example, the output looks like:

```
Preparing insert statement
Executing prepared insert
Inserting NULL values
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x00007ffa1ae42ca1, pid=14296, tid=13876
#
# JRE version: Java(TM) SE Runtime Environment (16.0+36) (build 16+36-2231)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (16+36-2231, mixed mode, sharing, tiered, compressed oops, g1 gc, windows-amd64)
# Problematic frame:
# C  [bat.dll+0x332ca1]
#
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
#
# An error report file with more information is saved as:
# C:\Projects\DBGL\hs_err_pid14296.log
#
# If you would like to submit a bug report, please visit:
#   https://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
```

**Comment by rcblanke** [comment]
```
---------------  S U M M A R Y ------------

Command Line: -ea -Dfile.encoding=Cp1252 -XX:+ShowCodeDetailsInExceptionMessages org.eclipse.jdt.internal.junit.runner.RemoteTestRunner -version 3 -port 53456 -testLoaderClass org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader -loaderpluginname org.eclipse.jdt.junit4.runtime -classNames org.dbgl.test.TemplateTests

Host: AMD Ryzen Threadripper 2950X 16-Core Processor , 32 cores, 31G,  Windows 10 , 64 bit Build 19041 (10.0.19041.804)
Time: Wed Apr 14 14:59:30 2021 West-Europa (zomertijd) elapsed time: 3.650327 seconds (0d 0h 0m 3s)

---------------  T H R E A D  ---------------

Current thread (0x00000177e287e7f0):  JavaThread "main" [_thread_in_native, id=11076, stack(0x00000027bfa00000,0x00000027bfb00000)]

Stack: [0x00000027bfa00000,0x00000027bfb00000],  sp=0x00000027bfafd110,  free space=1012k
Native frames: (J=compiled Java code, j=interpreted, Vv=VM code, C=native code)
C  [ntdll.dll+0x24785]
C  [AcLayers.DLL+0x904f]
C  [AcLayers.DLL+0x77bd]
C  [ucrtbase.dll+0xf05b]
C  [bat.dll+0x3075bc]

Java frames: (J=compiled Java code, j=interpreted, Vv=VM code)
j  org.monetdb.monetdbe.MonetNative.monetdbe_query(Ljava/nio/ByteBuffer;Ljava/lang/String;Lorg/monetdb/monetdbe/MonetStatement;ZI)Ljava/lang/String;+0
j  org.monetdb.monetdbe.MonetStatement.execute(Ljava/lang/String;)Z+38
j  org.monetdb.monetdbe.MonetConnection.executeCommand(Ljava/lang/String;)V+20
j  org.monetdb.monetdbe.MonetConnection.commit()V+24
```

**Comment by bernardom29** [comment]
This was fixed in c3e34f8422fc19e0ab995121dff828a539950cbb. 
I tried both examples mentioned here, and they both work.
